### DISCUSSION TITLE
Question regarding `path.files`

### DISCUSSION THREAD

**Discussion by ghost** [moudey/Shell#501] [General]
As per [the docs](https://github.com/moudey/Shell/blob/e58d0bd97a84f344aa4f0c07c590570d9fe2bdda/docs/functions/path.html#L140) of the function `path.files` :
```js
path.files(sys.dir, ["*"], flags[2=files | 3=dirs | 5=files+dirs | 8=quots | 16=full path], sep)
```
The flag `3` is for directories and `2` is for files. However the opposite seems to be true in my case:
```js
$folder_s=path.files(path.curdir,'*',2)
$file_s=path.files(path.curdir,'*',3)
item ( title='@(len(folder_s)) Folders and @(len(file_s)) Files' )
```
![test](https://github.com/user-attachments/assets/56655114-65fe-44b0-8d28-6bb550c3625c)

Just wanted to confirm if its a typo in the docs or not.

**Comment by RubicBG** [comment]
yes, the values ​​are swapped, but I don't know if it's a programming error or a documentation error
